Yoast/wordpress-seo
# P3 140 make SEMrush response more closely related

## Context


* The related keyphrases results should be ordered by "relevance" instead of "volume of searches"

## Summary


This PR can be summarized in the following changelog entry:

* Change the sort order in the related keyphrases result from "volume" to "relevance".

## Relevant technical choices:

* During the testing phase, due to a temporary SEMrush API issue, a bug was discovered in handling the errors in the response. This PR addresses this, by making sure that the keyphrases table is not tried to be rendered (thus crashing) when the response does not contain an array of results.

## Test instructions

This PR can be tested by following these steps:

* Edit a (new) post and insert a keyphrase
* open the modal and see that the results are not sorted anymore by volume, and the first ones should be more relevant

## UI changes

## Documentation

## Quality assurance


Fixes [P3-140]
